tests/unit/metadata_api/test_client.py

```diff
@@ -3,23 +3,37 @@
 
 from requests import Response
 
-get_projects_resp = {
-    "projects": [
-        "ioc-1000",
-        "ioc-1099",
-    ]
-}
+from tests.factories.metadata_api import TimeSeriesIdMetadataFactory
 
+get_projects_resp = {"projects": ["ioc-1000", "ioc-1099"]}
 
-def make_response(content: dict, status_code: int = 200, reason: str = "OK") -> Response:
+
+def make_response(
+    content: dict = None, dumped_data: str = None, status_code: int = 200, reason: str = "OK"
+) -> Response:
     response_obj = Response()
     response_obj.status_code = status_code
-    response_obj._content = json.dumps(content).encode()
+    response_obj._content = dumped_data if dumped_data else _dump_any_dict(content).encode()
     response_obj.reason = reason
     return response_obj
 
 
+def _dump_any_dict(target: dict) -> str:
+    return json.dumps(target, sort_keys=True, default=str)
+
+
 def test_get_projects(metadata_api_client):
     with patch.object(metadata_api_client, "get", return_value=make_response(content=get_projects_resp)):
         res = metadata_api_client.get_projects()
     assert res == ["ioc-1000", "ioc-1099"]
```

Comment:
```suggestion
def test_get_projects(metadata_api_client):
    projects = ["ioc-1000", "ioc-1099"]

    with patch.object(metadata_api_client, "get", return_value=make_response(content={"projects": projects})):
        res = metadata_api_client.get_projects()
    assert res == projects
```

That would both make the test explicit and would allow to get rid of the global mutable variable.

Reply:
Good point, thanks :+1: